### ISSUE TITLE
Sphereface64 just with softmax

### ISSUE THREAD

**Issue by zhangxinyu-xyz** [clcarwin/sphereface_pytorch#28]
Hi, I want to make a base model on sphereface64 just using softmax. I use webface but the model can't converage to low loss. The loss is about 4.5. And I use FC6 as classification layer.
Have you tryed to use softmax to train the model? Is there something wrong?
